aws_lambda_builders/workflows/custom_make/utils.py

```diff
@@ -35,6 +35,15 @@ def normpath(self, path):
     def abspath(self, path):
         return os.path.abspath(path)
 
+    def isabspath(self, path):
+        return os.path.isabs(path)
+
+    def relpath(self, path, start):
+        return os.path.relpath(path, start)
+
+    def join_relpath(self, path, start):
```

Comment:
do we need this join_relpath? we seem to be just joining paths and not relative paths.

Reply:
no it can be just join_path

Reply:
removed.